src/vs/workbench/electron-browser/workbench.ts

```diff
@@ -202,6 +204,7 @@ export class Workbench implements IPartService {
 	private editorsVisibleContext: IContextKey<boolean>;
 	private inZenMode: IContextKey<boolean>;
 	private hasFilesToCreateOpenOrDiff: boolean;
+	private fontAliasing: string;
```

Comment:
You can make this type: `private fontAliasing: 'default' | 'antialiased' | 'none'`